Packs/SEKOIAIntelligenceCenter/Integrations/SEKOIAIntelligenceCenter/SEKOIAIntelligenceCenter.py

```diff
@@ -166,13 +166,16 @@ def extract_indicator_from_pattern(stix_object: dict) -> str:
     return item[0][2].strip("'")
 
 
-def extract_file_indicator_hashes(pattern_str: str | None) -> dict:
+def extract_file_indicator_hashes(stix_object: str | None) -> dict:
     """
     Extract hashes composing the STIX indicator pattern
     """
     hashes = {"sha512": "", "sha256": "", "sha1": "", "md5": ""}
```

Comment:
```suggestion
    hashes = {"sha512": None, "sha256": None, "sha1": None, "md5": None}
```
We don't want to set an empty string to an indicator value.